generator/osm_translator.hpp

```diff
@@ -244,6 +245,99 @@ class RelationTagsWay : public RelationTagsBase
   }
 };
 
+class BusStopProcessor
+{
+public:
+  bool ProcessBusStop(OsmElement * p, FeatureParams & params)
+  {
+    ASSERT_EQUAL(p->type, OsmElement::EntityType::Node, ());
+
+    static string const kHighway = "highway";
+    static string const kBusStop = "bus_stop";
+
+    if (p->GetTag(kHighway) != kBusStop)
+      return false; // not a bus stop
+
+    auto i = m_busStops.find(p->id);
+    if (i != m_busStops.end())
+      return false; // bus already exists
+
+    BusStopInfo & bsi = m_busStops[p->id];
+    bsi.params = params;
+    bsi.pt = MercatorBounds::FromLatLon(p->lat, p->lon);
+
+    return true;
+  }
+
+  bool ProcessBusRoute(OsmElement * p)
+  {
+    ASSERT_EQUAL(p->type, OsmElement::EntityType::Relation, ());
+
+    static string const kRoute = "route";
```

Comment:
No need in static - move implementation to the cpp and create a couple of globals in the anonymous namespace.